intranet/static/js/eighth/schedule.js

```diff
@@ -175,5 +175,21 @@ $(function() {
         if (activities !== "" && !confirm("Are you sure you want to add the following activities without a sponsor?\n" + activities)) {
             e.preventDefault();
         }
+        var stickyCancel = "";
+
+        $("tr.form-row:not(.hidden)").each(function(i, el) {
+          if($(this).hasClass("scheduled")){
```

Comment:
```suggestion
        $("tr.form-row.scheduled:not(.hidden)").each(function() {
```

(Assuming the corresponding changes to the rest of the function, of course.)